lib/livebook_web/live/home_live.ex

```diff
@@ -18,12 +18,18 @@ defmodule LivebookWeb.HomeLive do
     sessions = Sessions.list_sessions()
     notebook_infos = Notebook.Learn.visible_notebook_infos() |> Enum.take(3)
 
+    # TODO:
+    # current_user = socket.assigns.current_user
+    # recently_opened_sessions = Sessions.list_recently_opened_sessions(current_user)
+    recently_opened_sessions = []
```

Comment:
How can I store recently sessions?
Tracker? :ets? GenServer?

Reply:
@ByeongUkChoi my suggestion is to have a GenServer to track this. It keeps a list of the last 10 sessions and stores it on Livebook.Storage, so the data is available when the server restarts. :)